documentation/app/template-remote-cluster.md

```diff
@@ -0,0 +1,187 @@
+## Bring your own cluster (remote deployments)
+
+This guide explains how to deploy application workloads to a remote OpenShift cluster while keeping Tekton Pipelines and webhooks on the host (RHDH) cluster. The GitOps application is split into two Argo CD apps:
+
+- app: Deploys the application manifests to the destination cluster (host or remote)
```

Comment:
I'm not sure here if all users understand fully the context of destination cluster (might be good to explain a bit more how argoCD works or share a link from their documentation where folks can look into for more details